docs/tutorials/tut-send-event-without-kyma-app.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Send events without a Kyma Application
+type: Tutorials
+---
+
+**In-cluster Eventing** allows publishers to send messages and subscribers to receive them without the need for a Kyma Application. This means that instead of the usual event flow where Application Connector publishes events to the Event Publisher Proxy, events can be published from within the cluster directly to the Event Publisher Proxy.
+
+
+## Prerequisites
+
+- A running Kyma cluster with a valid Namespace
+
+## Steps
+
+1. In Kyma's left navigation panel, go to **Workloads** > **Functions** and navigate to your Function.
+
+2. Once in the Function details view, Switch to the **Configuration** tab, and select **Create Event Subscription** in the **Event Subscriptions** section.
+
+3. Create a subscription where the **eventType.value** field includes the name of your Application. In this example, this is `sap.kyma.custom.nonexistingapp.order.created.v1`, where `nonexistingapp` is an Application that does not exist in Kyma.
+
+```yaml
+apiVersion: eventing.kyma-project.io/v1alpha1
+kind: Subscription
+metadata:
+  name: mysub
+  namespace: mynamespace
+spec:
+  filter:
+    filters:
+    - eventSource:
+        property: source
+        type: exact
+        value: ""
+      eventType:
+        property: type
+        type: exact
+        value: sap.kyma.custom.nonexistingapp.order.created.v1
+  protocol: ""
+  protocolsettings: {}
+  sink: http://myservice.mynamespace.svc.cluster.local
+
+```
+
+
+4. Select the event type and version that you want to use for your Function and select **Save** to confirm changes.
+
+The message appears on the UI confirming that the Event Subscription was successfully created, and you will see it in the **Event Subscriptions** section in your Function.
```

Comment:
```suggestion
You get a confirmation that the Event Subscription was successfully created, and you will see it in the **Event Subscriptions** section in your Function.
```

"...you will see **_it_**..." - will I see the message that appears or the Event Subscription? Context implies the subscription, but could be the confirmation message too, sentence is ambiguous.